roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -5,7 +5,7 @@ apiLevels:
 - v1
 apiVersion: v1
 assetConfig:
-  logoutURL: ""
+  logoutURL: {{ openshift.master.logout_url }}
```

Comment:
Sorry by the question but, isn't it enough:

<pre>
logout_url: "{{ openshift_master_logout_url | default(None) }}"
</pre>

I've done it at roles/openshift_master/tasks/main.yml